tests/unit/TagFieldTest.php

```diff
@@ -0,0 +1,395 @@
+<?php
+
+class TagFieldTest extends SapphireTest {
+	/**
+	 * @var string
+	 */
+	public static $fixture_file = 'tagfield/tests/unit/TagFieldTest.yml';
+
+	/**
+	 * @var array
+	 */
+	protected $extraDataObjects = array(
+		'TagFieldTest_BlogTag',
+		'TagFieldTest_BlogPost',
+	);
+
+	/**
+	 * @test
+	 */
+	public function itSavesLinksToExistingTagsOnExistingRecords() {
```

Comment:
Might be nice to discuss as part of a larger unit test style guide. I don't think it matters either way...